respy/tests/test_conditional_draws.py

```diff
@@ -12,16 +12,17 @@
 from respy.config import TEST_RESOURCES_DIR
 
 
-def fixture():
+@pytest.fixture
+def kalman_results():
     """The inputs and outputs were generated using a well tested Kalman filter."""
     with open(TEST_RESOURCES_DIR / "conditional_draws_fixture.pickle", "rb") as p:
         fix = pickle.load(p)
     return fix
 
 
 @pytest.mark.parametrize("i", range(20))
-def test_update_and_evaluate_likelihood(i):
-    fix = fixture()
+def test_update_and_evaluate_likelihood(i, kalman_results):
+    fix = kalman_results
```

Comment:
This line is not necessary. If you are concerned about changing ``kalman_results`` use ``pytest.fixture(scope="function")`` which recreates the output of the fixture for every test function. You can also use module or others.

Reply:
done